src/js/rx/reducers/disclaimer.js

```diff
@@ -0,0 +1,18 @@
+import set from 'lodash/fp/set';
+
+const initialState = {
+  visible: true
+};
+
+export default function disclaimer(state = initialState, action) {
+  switch (action.type) {
+    case 'OPEN_DISCLAIMER':
+      return Object.assign({}, state, {
```

Comment:
I think it's safe to remove that action entirely. It will be open by default, and once it's closed we don't need to reopen it.